compiler/lsp-tests/src/Main.hs

```diff
@@ -730,9 +725,26 @@ hTakeUntil handle regex = go
           line <- hGetLine handle
           if pred line then pure (Just line) else go
 
+assertUntilWithout :: Handle -> T.Text -> T.Text -> IO (Maybe String)
```

Comment:
Mind just adding a single line comment what this does? I had to read the whole function to get the intuition of what it's supposed to do.

Something like this:
-- Reads until the first argument of text is present and is successful only when it finds this without finding the second text argument.

Reply:
Fixed - I've added a slightly longer comment with an example.